src/index.ts

```diff
@@ -13,5 +18,18 @@ export const storagePlugin: Plugin = {
     evaluators: [],
     providers: [],
 };
+
+export const getStorageClient = async (runtime: IAgentRuntime): Promise<StorageClientInstanceImpl> => {
```

Comment:
Probably good to add some jsdoc comment here to indicate that this will return the first client that has been created.

Possibly better, return an array of clients so you don't have to throw if there are none, and if more than one has been created they can be accessed.

```suggestion
export const getStorageClients = async (runtime: IAgentRuntime): Promise<StorageClientInstanceImpl[]> => {
```

Its a trivial change to the caller because if they want the first they just do:

```js
const [client] = await getStorageClients(runtime)
```

Reply:
@alanshaw I've added the jsdoc comment, but didn't change it to return a list because we need to find the client named `storage`, so we can cast that and return that instance only.